src/irc/core/irc.c

```diff
@@ -158,6 +159,11 @@ static char *split_nicks(const char *cmd, char **pre, char **nicks, char **post,
 
 	*pre = g_strdup(cmd);
 	*post = *nicks = NULL;
+
+	if (**pre == '@') { /* message-tags */
+		arg++;
```

Comment:
This one took me a while to understand, could use a longer comment

Reply:
I added a comment

Reply:
Sounds good